requirements.txt

```diff
@@ -8,6 +8,7 @@ lazy-object-proxy==1.6.0
 mccabe==0.6.1
 psycopg2-binary==2.9.1
 pylint==2.9.5
+pylint-django==2.4.4
```

Comment:
Don't forget to configure pylint to actually load this plugin. It is currently not used in your setup.